Answers.md

```diff
@@ -1,5 +1,14 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
+    server side routing makes a request that will reload a different page for each clicked link, client side is handled interally and the request to the server is prevented, a new component is loaded instead
```

Comment:
It is not really "prevented" as You can still request the server for data, but the page won't refresh because Javascript will take care of it internally with life cycle components and state. For example You can reload data from server via axios request when component is updated and page won't refresh. You can also lookup interesting things like Hot Reloading.